views/index.hbs

```diff
@@ -45,6 +45,32 @@
             </div>
         </div>
 
+		<div class="row">
+			<div class="col-md-6">
+				<h3 class="text-center">Sortie</h3>
+				<ul class="list-group">
+					<li class="list-group-item list-group-item-borderless"><b id="sortieBoss"></b> | <span id="sortieFaction"></span></li>
+					<div id="sortieList">
+						<li class="list-group-item list-group-item-borderless" id="sortiebody">
+							<span id="sortietitle">Loading Sortie...</span>
+						</li>
+					</div>
+					<li class="list-group-item list-group-item-borderbottom" id=""></li>
+				</ul>
+			</div>
+
+			<div class="col-md-6">
+				<h3 class="text-center">Void Fissures</h3>
+
+				<ul class="list-group-item list-group-item-borderless">
```

Comment:
The unlabeled list should have the class list-group instead of the list item classes.